packages/caliper-core/lib/config/config-util.js

```diff
@@ -59,9 +59,14 @@ const keys = {
     ZooAddress: 'caliper-zooaddress',
     ZooConfig: 'caliper-zooconfig',
     TxUpdateTime: 'caliper-txupdatetime',
-    SkipStartScript: 'caliper-skipstartscript',
-    SkipEndScript: 'caliper-skipendscript',
     Logging: 'caliper-logging',
+    Flow: {
+        StartOnly: 'start-only',
```

Comment:
1. Should prefix these with `caliper-` to follow the convention. 
2. The key hierarchy results in the following names: `caliper-flow-start-only`, etc. (So, missing `flow-` prefix)